Makefile

```diff
@@ -281,6 +290,9 @@ $(CONFIG): $(CONFIG_IN) $(GENERATED_FILES)
 		-e "s|@DEFVCPUS@|$(DEFVCPUS)|g" \
 		-e "s|@DEFMEMSZ@|$(DEFMEMSZ)|g" \
 		-e "s|@DEFDISABLEBLOCK@|$(DEFDISABLEBLOCK)|g" \
+		-e "s|@DEFENABLEMEMPREALLOC@|$(DEFENABLEMEMPREALLOC)|g" \
+		-e "s|@DEFENABLEMSWAP@|$(DEFENABLESWAP)|g" \
+		-e "s|@DEFENABLEMSWAP@|$(DEFENABLEDEBUG)|g" \
```

Comment:
Hi @mcastelino - In passing, I've just spotted the above, which won't work as expected I think.

Reply:
@jodh-intel thanks for catching this. This is a bug. I will send a PR to address this.